Template/Trigger/ElementVisibilityTrigger.web.js

```diff
@@ -310,9 +273,122 @@
             return false;
         }
 
+        function setMutationObserver(triggerEvent) {
+            return function () {
+                if (observeDomChanges && isMutationObserverSupported) {
+                    var targetNode = documentAlias.body;
+                    var config = {attributes: true, childList: true, subtree: true};
+                    observerMutation = new MutationObserver(function (mutationsList) {
+                        Array.prototype.push.apply(allMutationsList, mutationsList);
+                        if (mutationObseverTimeout) {
+                            return;
+                        }
+
+                        mutationObseverTimeout = true;
+                        setTimeout(function () {
+                            mutationObserverCallback(allMutationsList, triggerEvent);
+                            allMutationsList = [];
+                            mutationObseverTimeout = false;
+                        }, 120);
+                    });
+
+                    observerMutation.observe(targetNode, config);
```

Comment:
not too important, here we could directly use `documentAlias.body`

Reply:
Updated as suggested